DPPP/test/unit/tPSet.cc

```diff
@@ -361,24 +395,31 @@ BOOST_AUTO_TEST_SUITE(pset)
 
 using DP3::DPPP::TestPSet;
 
-BOOST_AUTO_TEST_CASE( test_none ) {
-  TestPSet::testNone();
-}
+BOOST_AUTO_TEST_CASE( test_none ) { TestPSet().testNone(); }
 
-BOOST_AUTO_TEST_CASE( test_bl ) {
-  TestPSet::testBL();
-}
+BOOST_AUTO_TEST_CASE( test_bl1 ) { TestPSet().testBL1(); }
```

Comment:
Cool you already fixed that :). Would it be possible to not have the tests in the class but just as unit tests. We should do that everywhere, but that is too big of a change but since we are reworking this one already we might as well do it